sub-logs/arnav.json

```diff
@@ -0,0 +1,15 @@
+{
+    "owner": {
+        "username": "arnavravinder",
+        "email": "arnavravinder2009@gmail.com"
+    },
+
+    "target": {
+        "TXT": {
+            "name": "arnav",
+            "value": "vc-domain-verify=arnav.is-a-good.dev,f26d3b202451267b81c8"
+        }
+    },
```

Comment:
TXT records should be followed by additional records.

Reply:
What record do you need?

Reply:
CNAME or A or AAAA is permissible.